PocketKit/Sources/PocketKit/Collections/CollectionViewModel.swift

```diff
@@ -77,9 +77,15 @@ class CollectionViewModel: NSObject {
         collectionController.delegate = self
         buildActions()
 
-        item?.savedItem?.publisher(for: \.isFavorite).sink { [weak self] _ in
+        item?.publisher(for: \.savedItem?.isFavorite).sink { [weak self] _ in
             self?.buildActions()
         }.store(in: &collectionItemSubscriptions)
+
+        item?.publisher(for: \.savedItem).sink { [weak self] _ in
+            self?.buildActions()
+        }.store(in: &collectionItemSubscriptions)
```

Comment:
actions were not being updated properly after a user saves a collection that was a recommendation